media.go

```diff
@@ -137,6 +137,16 @@ func getname(name string) string {
 	return name
 }
 
+func getnameoverwrite(name string) string {
```

Comment:
It must be more readable.
like `getNameOverwrite`.

Also I think it's better to change name of the file and add a simple number like unix time.

Reply:
Sounds good. I will take care of this as soon as I get some time.